test/test_fourier.jl

```diff
@@ -0,0 +1,419 @@
+        out_ev = avg_pds_from_events(
+            times,
+            gti,
+            segment_size,
+            dt,
+            norm=norm,
+            use_common_mean=use_common_mean,
+            silent=true,
+            fluxes=nothing,
+        )
+        out_ct = avg_pds_from_events(
+            bin_times,
+            gti,
+            segment_size,
+            dt,
+            norm=norm,
+            use_common_mean=use_common_mean,
+            silent=true,
+            fluxes=counts,
+            errors=errs,
+        )
+        # The variance is not _supposed_ to be equal, when we specify errors
+        if use_common_mean
+            compare_tables(out_ev, out_ct, rtol=0.01, discard=["variance"])
+        else
+            compare_tables(out_ev, out_ct, rtol=0.1, discard=["variance"])
+        end
+    end
+
+    @testset "test_avg_cs_cts_and_events_are_equal" for use_common_mean in [true,false], norm in ["frac", "abs", "none", "leahy"]
+        out_ev = avg_cs_from_events(
+            times,
+            times2,
+            gti,
+            segment_size,
+            dt,
+            norm=norm,
+            use_common_mean=use_common_mean,
+            silent=true,
+        )
+        out_ct = avg_cs_from_events(
+            bin_times,
+            bin_times,
+            gti,
+            segment_size,
+            dt,
+            norm=norm,
+            use_common_mean=use_common_mean,
+            silent=true,
+            fluxes1=counts,
+            fluxes2=counts2,
+        )
+        if use_common_mean
+            compare_tables(out_ev, out_ct, rtol=0.01)
+        else
+            compare_tables(out_ev, out_ct, rtol=0.1)
+        end
+    end
+
+    @testset "test_avg_cs_cts_and_err_and_events_are_equal" for use_common_mean in [true,false], norm in ["frac", "abs", "none", "leahy"]
+        out_ev = avg_cs_from_events(
+            times,
+            times2,
+            gti,
+            segment_size,
+            dt,
+            norm=norm,
+            use_common_mean=use_common_mean,
+            silent=true,
+        )
+        out_ct = avg_cs_from_events(
+            bin_times,
+            bin_times,
+            gti,
+            segment_size,
+            dt,
+            norm=norm,
+            use_common_mean=use_common_mean,
+            silent=true,
+            fluxes1=counts,
+            fluxes2=counts2,
+            errors1=errs,
+            errors2=errs2,
+        )
+        discard = [m for m in propertynames(out_ev) if m == :variance]
+
+        if use_common_mean
+            compare_tables(out_ev, out_ct, rtol=0.01, discard=discard)
+        else
+            compare_tables(out_ev, out_ct, rtol=0.1, discard=discard)
+        end
+    end
+end
+
+@testset "test_norm" begin
+    mean = var = 100000
+    N = 1000000
+    dt = 0.2
+    meanrate = mean / dt
+    lc = rand(Poisson(mean),N)
+    pds = abs2.(fft(lc))
+    freq = fftfreq(N, dt)
+    good = 2:floor(Int,N/2)
```

Comment:
This is always integer division:
```suggestion
    good = 2:(N ÷2)
```